tests/test_json_encoder.py

```diff
@@ -22,60 +22,60 @@ def test_encodes_records_correctly(self):
         self.assertJsonEqual(
             """
             {
-              "timestamp": "20170319T150123Z",
+              "timestamp": "20170319T1501Z",
               "soil_moisture": 305
             }
             """.strip(),
             self.encoder.encode(
                 db_store.SoilMoistureRecord(
                     timestamp=datetime.datetime(
-                        2017, 3, 19, 15, 1, 23, 924000, tzinfo=pytz.utc),
+                        2017, 3, 19, 15, 1, 0, tzinfo=pytz.utc),
```

Comment:
I was thinking for equivalence to what we do in the poller tests, but now that you mention it, it doesn't make sense because the poller tests are because the poller generates timestamps with 0.